vcd/resource_vcd_org_vdc.go

```diff
@@ -536,6 +600,152 @@ func resourceVcdVdcDelete(d *schema.ResourceData, meta interface{}) error {
 	return nil
 }
 
+// updateAssignedVmSizingPolicies handles VM sizing policies.
+func updateAssignedVmSizingPolicies(vcdClient *VCDClient, d *schema.ResourceData, meta interface{}) error {
+	if vcdClient.Client.APIVCDMaxVersionIs(">= 33.0") {
+		log.Printf("[TRACE] updating assigned VM sizing policies to VDC")
+
+		_, idsOk := d.GetOk("vm_sizing_policy_ids")
+		_, defaultIdOk := d.GetOk("default_vm_sizing_policy_id")
+
+		if idsOk != defaultIdOk {
```

Comment:
Just thinking. This will "continue" even if both fields are not set. Looks like it will still be no-op in later conditions, but maybe we should exit early if none are set?

Reply:
look ok, cause later it uses d.HasChange to use values or not

Reply:
add early return